go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/firecracker-microvm/firecracker-go-sdk
 
-go 1.18
+go 1.21
```

Comment:
any reason we are useing 1.21 and not 1.22

Reply:
I bumped it to match the minimum version requirements of our dependencies

Reply:
As this is a library I think it's good practice to use the minimum possible version